scripts/pytest-tests.sh

```diff
@@ -30,24 +34,18 @@ cd "$SRCDIR/test/python" && source ./venv/bin/activate
 
 run_tests << 'END'
 
-test_bdd_pool
-test_bdd_replica
-test_bdd_nexus
-test_nexus_publish
-test_bdd_rebuild
-test_bdd_csi
-test_csi
-
-test_ana_client
-test_cli_controller
-test_replica_uuid
-# test_rpc
-
-test_bdd_nexus_multipath
-test_nexus_multipath
-
-test_multi_nexus
-test_nexus
-test_null_nexus
+tests/replica
+tests/publish
+tests/rebuild
+
+# tests/csi
```

Comment:
are the CSI tests disabled now?

Reply:
Yes, but this is a temporary measure while I resolve issues with running the mayastor-csi plugin from within a container.